.travis.yml

```diff
@@ -45,6 +46,16 @@ before_script:
 
 # Generate documentation, compile the engine, run tests.
 script: |
+  # Check if cargo-update is present
+  if ![cargo install-update --version]; then
```

Comment:
Could simplify to `which appname || install`.

Reply:
This has complicated semantics with cargo subcommands so I prefer to just check if the command works.